momentum_scanner_intraday.py

```diff
@@ -81,7 +81,7 @@ def get_high_returns(
     # ]
     if df.empty:
         return df
-    df = df[df["24H Return"] >= 0]
+    #df = df[df["24H Return"] >= 0]
```

Comment:
Possibly drop a comment here about why this is commented out instead of just not here.

Reply:
Dropped a comment